repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -225,9 +313,9 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
       configuredMethods.contains(HttpMethod.fromValue(requestMethod))
 
   private def getPathsForContentType(contentType: String, resource: HttpMessage): List[LinkPath] = {
-    Option(contentType) match {
-      case Some(ct) if ct.toLowerCase.contains("json") => resource.getJson.toList
-      case Some(ct) if ct.toLowerCase.contains("xml") => resource.getXml.toList map(_.getXpath)
+    (Option(contentType), Option(resource)) match {
```

Comment:
Were you having some issue with resource being null? If so that seems like a testing problem, because I'm fairly certain it's not possible in usage. Of course the diff doesn't give me enough context to say for certain.